docs/Overview/SupportedDataSources.md

```diff
@@ -9,81 +9,82 @@ sidebar_position: 5
+| Jenkins          | CI/CD                                                                        | 2.263.x+                           | Available               | `jenkins`                      | Incremental Sync                                                |
+| BitBucket (Beta) | Source Code Management, Code Review                                          | Cloud                              | Advanced Mode Available | `bitbucket`, `gitextractor`    | Full Refresh                                                    |
+| TAPD (Beta)      | Issue Tracking                                                               | Cloud                              | Advanced Mode Available | `tapd`                         | Full Refresh, Incremental Sync(for `stories`, `bugs`, `tasks`)  |
+| Zentao (Beta)    | Issue Tracking                                                               | v17.x, v18.x                       | Advanced Mode Available | `zentao`                       | Full Refresh                                                    |
+| Gitee (WIP)      | Source Code Management, Code Review, Issue Tracking                          | Cloud                              | Not Available           | `gitee`, `gitextractor`        | Full Refresh, Incremental Sync(for `issues`,`MRs`)              |
+| PagerDuty (WIP)  | Issue Tracking                                                               | Cloud                              | Not Available           | `pagerduty`                    | Full Refresh                                                    |
+| Feishu (WIP)     | Calendar                                                                     | Cloud                              | Not Available           | `feishu`                       | Full Refresh                                                    |
+| AE               | Source Code Management                                                       | On-prem                            | Not Available           | `ae`                           | Full Refresh                                                    |
+| Sonarqube        | CODE QUALITY                                                                 | SonarQube v8.x, v9.x               | Available               | `sonarqube`                    | Full Refresh                                                    |
+| Bamboo CI(WIP)   | CI/CD                                                                        | v6.8.1                             | Not Available           | `bamboo`                       | Full Refresh                                                    |
+| Bamboo CI(WIP)   | CI/CD, Source Code Management, Code Review                                   | Cloud                              | Available               | `azuredevops`, `gitextractor`  | Full Refresh                                                    |
```

Comment:
@CamilleTeruel Seems like you meant "Azure DevOps (WIP)" instead of "Bamboo CI (WIP)" here.